rasa/utils/tensorflow/layers.py

```diff
@@ -121,6 +126,33 @@ def __init__(self, reg_lambda: float = 0, **kwargs: Any) -> None:
 
         super().__init__(kernel_regularizer=regularizer, **kwargs)
 
+    def get_units(self) -> int:
+        """Returns number of output units."""
+        return self.units
+
+    def get_kernel(self) -> tf.Tensor:
+        """Returns kernel tensor."""
+        return self.kernel
+
+    def get_bias_info(self) -> Tuple[bool, tf.Tensor]:
+        """Returns whether the layer has a bias tensor and the bias tensor itself."""
+        return self.use_bias, self.bias
```

Comment:
Do you need the bias if `self.use_bias` is set to `False?
If not, then you can write this as - 
```
if self.use_bias:
    return self.bias
return None
```
Reduces what all you need to return and the function name can become `get_bias`.